test/src/main/java/org/apache/accumulo/test/functional/BulkNewIT.java

```diff
@@ -508,6 +532,26 @@ public void testEndOfFirstTablet() throws Exception {
 
   @Test
   public void testManyFiles() throws Exception {
+
+    // Metrics collector Thread
+    final LinkedBlockingQueue<TestStatsDSink.Metric> queueMetrics = new LinkedBlockingQueue<>();
+    final AtomicBoolean shutdownTailer = new AtomicBoolean(false);
```

Comment:
Should something eventually set this to true?

Reply:
Yes, the tailer is shutdown in the new version of this test.